src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -8872,6 +8919,20 @@ void Assembler::vextractf64x4(Address dst, XMMRegister src, uint8_t imm8) {
   emit_int8(imm8 & 0x01);
 }
 
+void Assembler::evextractps(Register dst, XMMRegister src, uint8_t imm8) {
+  assert(VM_Version::supports_avx(), "");
+  assert(imm8 <= 0x03, "imm8: %u", imm8);
+  InstructionAttr attributes(AVX_128bit, /* vex_w */ true, /* legacy_mode */ false, /* no_mask_reg */ true, /* uses_vl */ true);
+  attributes.set_is_evex_instruction();
```

Comment:
assert should check accordingly for SSE4_1 and above.
uses_vl should be false. 
set_is_evex_instruction is not needed.